code/_globalvars/lists/flavor_misc.dm

```diff
@@ -2,7 +2,9 @@
 	//Head accessory styles
 var/global/list/head_accessory_styles_list = list() //stores /datum/sprite_accessory/head_accessory indexed by name
 	//Marking styles
-var/global/list/marking_styles_list = list() //stores /datum/sprite_accessory/horns indexed by name
+var/global/list/marking_styles_list = list() //stores /datum/sprite_accessory/body_markings indexed by name
+	//Organs
+var/global/list/organs_data = list() //Stores organs that have been modified via preferences (amputations, prosthetics).
```

Comment:
Organs_data is a globalized version of organ_data from preferences so adjusted organs can be checked and decisions made based on that information

For example, code/game/objects/items/weapons/scissors.dm line 39 where it checks to see if they've got the default Morpheus head (has a screen), thus no facial hair

Another example is the screen/optic adjustment proc, where it decides which type of customization to do based on whether the head is morpheus (screen) or not (optics)